arch/archs.yml

```diff
@@ -1,4 +1,19 @@
 archs:
-  - name: empty
-    folder: empty
-    comment: initial file, no arch v2 defined.
+  - name: arc
+    folder: arc
+  - name: arm
+    folder: arm
+  - name: arm64
+    folder: arm64
+  - name: mips
+    folder: nios2
```

Comment:
Now you're not going to slip this one past the ISA!

Reply:
wonder what happened there

Reply:
Fixed.